user-contract-mon/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+
+	monitor "github.com/Eslam-Nawara/user-contract-mon/internal"
+	"github.com/NicoNex/echotron/v3"
+)
+
+func main() {
+	mon, err := monitor.NewMonitor("./.config")
```

Comment:
instead of hardcoding the config file, we can pass it as an argument and use default if not provided. using `flag` package for example